### ISSUE TITLE
[Reproducibility Issue] - Not able to achieve higher than 25% score (Paper & arc-prize)

### ISSUE THREAD

**Issue by ai-nikolai** [da-fr/arc-prize-2024#7]
Dear @da-fr and @InsanityCheck ,

Thanks for the great work again. We have an issue of reproducing your scores from your paper.

We re-ran your full-Nemo training script with a small adaptation **following your paper,** where we only train on a random subset of 300/400 (i.e. 75%) of the public eval data and use the remaining 100/400 (i.e. 25%) as "real" held-out test set. (This is the same setup as in your paper.)

Following a full training run and then a full test time training run with all the augmentations and scoring; we only achieve a score of around 25%. There is nothing more we can do on our end.

----
Please let us know how we can replicate your paper results (which indicate 50% or 60%+ on a similar setup).

**Ideally:**
1. Could you release the dataset from your paper (i.e. the exact 25% / 75% split you used).
2. Could you release the final pre-trained model from your paper trained on the 75% dataset (such that we can run it with your provided TTT code?)

Thank you.
